### PR TITLE
docs: Added example for GuildChannel#clone()

### PR DESCRIPTION
**Pull request by Strikerrr**
**Please describe the changes this PR makes and why it should be merged:**
This PR adds an example for GuildChannel#clone()

I think this is useful because this so far has not been documented and it would be useful for cloning channels in servers (idk what more to say)

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/structures/GuildChannel.js [hunk 1]
```diff
@@ -444,7 +444,19 @@ class GuildChannel extends Channel {
...
+   *  name: 'Some name',
+   *  withPermissions: true,
+   *  withTopic: false,
+   *  nsfw: false,
+   *  parent: this.parent,
+   *  reason: 'Some reason'
+   * })
+   *  .then(() => console.log('Channel cloned!'))
+   *  .catch(console.error)
+  */
```
Comment (KingDGrizzle) [inline src/structures/GuildChannel.js]: This looks like its missing a space :P

Comment (Lewdcario) [reply src/structures/GuildChannel.js]: Missing semi

Comment (Strikerrr) [reply src/structures/GuildChannel.js]: Added missing semi

File: src/structures/GuildChannel.js [hunk 2]
```diff
@@ -444,6 +444,18 @@ class GuildChannel extends Channel {
    * @param {ChannelResolvable} [options.parent=this.parent] The parent of the new channel
    * @param {string} [options.reason] Reason for cloning this channel
    * @returns {Promise<GuildChannel>}
+   * @example
+   * // Clone a channel
+   * channel.clone({
+   *  name: 'Some name',
```
Comment (Lewdcario) [inline src/structures/GuildChannel.js]: Didn't notice earlier: channels can't have spaces (we reject on this).

Comment (Strikerrr) [reply src/structures/GuildChannel.js]: Fixed

File: src/structures/GuildChannel.js [hunk 3]
```diff
@@ -444,6 +444,18 @@ class GuildChannel extends Channel {
...
    * @param {string} [options.reason] Reason for cloning this channel
    * @returns {Promise<GuildChannel>}
+   * @example
+   * // Clone a channel
+   * channel.clone({
+   *  name: 'Some-name',
+   *  withPermissions: true,
+   *  withTopic: false,
+   *  nsfw: false,
+   *  parent: this.parent,
```
[the remark this answers is not in the document]
Comment (Strikerrr) [reply src/structures/GuildChannel.js]: Will remove it

**Comment by iCrawl** [comment]
This won't do. Properly rebase or close reopen. Maybe pour some hours into learning git.

**Pull request closed without merging**